### ISSUE TITLE
Provide a selector for pause/play container

### ISSUE THREAD

**Issue by influencebydesign** [Accessible360/accessible-slick#79]
I'd like to be able to provide a container to place the pause/play button inside as I can do for the dots and arrows with the append options.

**Comment by sayvil** [comment]
I agree. I'm using appendArrows and appendDots and the pause button shows in an odd position.
